js-exercises/sum-fibs/sumFibs.js

```diff
@@ -0,0 +1,59 @@
+const isNumber = (arg) => typeof arg === 'number';
+
+const isOddNumber = (arg) => isNumber(arg) && (arg % 2 !== 0);
+
+const isPositiveNumber = (arg) => isNumber(arg) && (arg >= 0);
+
+function* returnNumberGenerator(number) {
```

Comment:
Why this? Since you've used this in only one place.

Reply:
It was cool for me to use `destructuring assignment` with help of generator.
Now it's not making that much sense, and I will remove this.

Reply:
Removed with 8ab346c55474d57acbd63b0f97940c49cb53006b